src/instructlab/configuration.py

```diff
@@ -205,7 +214,10 @@ class _chat(BaseModel):
         default=1.0,
         description="Controls the randomness of the model's responses. Lower values make the output more deterministic, while higher values produce more random results.",
     )
-    rag: _rag
+    rag: _rag = Field(
```

Comment:
Will the same `_rag` class be used also for the commands in the `rag` command group?

Reply:
No - renaming all the new ones to have prefix `_chat_rag`.

Reply:
Done - thanks!